utils/xxhash/Makefile

```
@@ -0,0 +1,93 @@
+#
+# Copyright (C) 2022 Julien Malik <julien.malik@paraiso.me>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+# initial implementation kindly stolen from (and then updated) :
+# https://forum.openwrt.org/t/xxhash-makefile/118426
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=xxhash
+PKG_VERSION:=0.8.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)_$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/Cyan4973/xxHash/archive/v$(PKG_VERSION)
+PKG_HASH:=3bb6b7d6f30c591dd65aaaff1c8b7a5b94d81687998ca9400082c739a690436c
+
+# The source for the library (xxhash.c and xxhash.h) is BSD
+# The source for the command line tool (xxhsum.c) is GPLv2+
+PKG_LICENSE:=BSD-2-Clause,GPL-2.0-or-later
+PKG_LICENSE_FILES:=LICENSE,cli/COPYING
+PKG_MAINTAINER:=Julien Malik <julien.malik@paraiso.me>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/xxhash/Default
+  TITLE:=Extremely fast hash algorithm
+  URL:=http://www.xxhash.com
```

Since this domain is redirected to GitHub, I would go for it.

```suggestion
  URL:=https://cyan4973.github.io/xxHash/
```